scripts/setup-windows.sh

```
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Stock Charts - Windows Development Environment Setup (Git Bash)
+# Assumes Git Bash for Windows is installed
+
+NODE_VERSION="24.13.1"
+PNPM_VERSION="10.29.3"
```

Shellcheck (0.11.0)

[warning] PNPM_VERSION appears unused. Verify use (or export if used externally).

(SC2034)